apps/dash-stitching/app.py

```diff
@@ -0,0 +1,495 @@
+import numpy as np
+import pandas as pd
+from skimage import io, data, transform
+from time import sleep
+
+import dash
+from dash.exceptions import PreventUpdate
+from dash.dependencies import Input, Output, State
+import dash_html_components as html
+import dash_core_components as dcc
+import dash_table
+
+import dash_canvas
+from dash_canvas.components import image_upload_zone
+from dash_canvas.utils import (
+    image_string_to_PILImage,
+    array_to_data_url,
+    parse_jsonstring_line,
+    brightness_adjust,
+    contrast_adjust,
+)
+from registration import register_tiles
+from utils import StaticUrlPath
+import pathlib
+
+app = dash.Dash(
+    __name__, meta_tags=[{"name": "viewport", "content": "width=device-width"}]
+)
+server = app.server
+app.config.suppress_callback_exceptions = True
+
+# get relative data folder
+PATH = pathlib.Path(__file__).parent
+
+DATA_PATH = PATH.joinpath("data").resolve()
+
+
+def demo_explanation(demo_mode):
+    if demo_mode:
```

Comment:
Since `demo_explanation(True)` is always going to be true, you don't need the `demo_mode` param and the if statement. You can just call `demo_explanation()` as a function in line 485

Reply:
fixed!